onedal/primitives/kernel_functions.py

```diff
@@ -131,7 +132,7 @@ def sigmoid_kernel(X, Y=None, gamma=1.0, coef0=0.0, queue=None):
     kernel_matrix : ndarray of shape (n_samples_X, n_samples_Y)
     """
 
-    X, Y, fptype = _check_inputs(X, Y)
-    return _compute_kernel({'fptype': fptype, 'method': 'dense',
+    X, Y = _check_inputs(X, Y)
+    return _compute_kernel({'method': 'dense',

```

Comment:
Why type is removed?

Reply:
`fptype` is not deleted but moved to this [location](https://github.com/KulikovNikita/scikit-learn-intelex/blob/174f2a1422c675f7da66271a2045aa7b1a1e6171/onedal/primitives/kernel_functions.py#L36)